pkgs/build-support/buildenv/builder.pl

```diff
@@ -117,10 +117,22 @@ sub addPkg {
     }
 }
 
+# Read packages list.
+my $pkgs;
+
+if (exists $ENV{"pkgsPath"}) {
+  local $/ = undef;
+  open FILE, $ENV{"pkgsPath"};
+  binmode FILE;
+  $pkgs = <FILE>;
+  close FILE;
+} else {
+  $pkgs = $ENV{"pkgs"}
+}
```

Comment:
Can you make the indentation consistent with the rest of the file?

I don't think `binmode` or setting `$/` are necessary here.